content/en/docs/instrumentation/java/libraries.md

```diff
@@ -23,3 +24,165 @@ some instrumentation libraries for your dependencies standalone. To find out
 which standalone instrumentation libraries are available, take a look at
 [this list](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
 Follow the instructions of each instrumentation library to set them up.
+
+## Example app preparation {#example-app}
```

Comment:
I'd suggest dropping the custom header anchors here and elsewhere in the added text. They aren't required, and it'll keep the markdown clean.